cmd/duffle/relocate.go

```diff
@@ -0,0 +1,266 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/pivotal/image-relocation/pkg/image"
+	"github.com/pivotal/image-relocation/pkg/pathmapping"
+	"github.com/pivotal/image-relocation/pkg/registry"
+
+	"github.com/deislabs/duffle/pkg/bundle"
+	"github.com/deislabs/duffle/pkg/duffle/home"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/validation"
+)
+
+const relocateDesc = `
+Relocates any docker and oci images referenced by a bundle, tags and pushes the images to a registry, and creates a new
+bundle with an updated image map.
+
+The --repository-prefix flag determines the repositories for the relocated images.
+Each image is tagged with a name starting with the given prefix and pushed to the repository.
+
+For example, if the repository-prefix is example.com/user, the image istio/proxyv2 is relocated
+to a name starting with example.com/user/ and pushed to a repository hosted by example.com.
+`
+
+type relocateCmd struct {
+	// args
```

Comment:
We don't usually have comments to separatte args, flags and other things here, but don't really have anything against them -- and might actually start using them in other places as well.